src/openrct2/Cheats.h

```diff
@@ -47,6 +47,7 @@ struct CheatsState
     bool AllowSpecialColourSchemes;
     bool MakeAllDestructible;
     StaffSpeedCheat SelectedStaffSpeed;
+    int ForcedParkRating;
```

Comment:
```suggestion
    int32_t ForcedParkRating;
```
Idem for the constant. We never use plain `int`.

Reply:
No, just the one you added. There rest is indeed better done in another PR.